catalogs/catalog-fileset/src/main/java/org/apache/gravitino/catalog/fileset/FilesetCatalogOperations.java

```diff
@@ -447,6 +446,11 @@ public Fileset createMultipleLocationFileset(
 
           filesetPathsBuilder.put(entry.getKey(), formalizePath);
           FileSystem fs = getFileSystemWithCache(formalizePath, conf);
+
+          if (fs.exists(formalizePath) && fs.getFileStatus(formalizePath).isFile()) {
+            throw new RuntimeException("Fileset location cannot be a file: " + formalizePath);
+          }
```

Comment:
should we also check the location of catalog/schema when creating a catalog/schema

Reply:
ok